recipe/meta.yaml

```diff
@@ -1,22 +1,22 @@
-{% set year = '2018' %}
-{% set version = year + '0712' %}
-{% set next_year = year|int + 1 %}
+{% set soname = '152' %}
```

Comment:
You may want to add a test to see if the same .soname version is built.

Reply:
Good point, will do.